ReactCommon/react/renderer/components/view/Touch.cpp

```diff
@@ -27,11 +27,11 @@ std::vector<DebugStringConvertibleObject> getDebugProps(
       {"target", getDebugDescription(touch.target, options)},
       {"force", getDebugDescription(touch.force, options)},
       {"timestamp", getDebugDescription(touch.timestamp, options)},
-      {"button", getDebugDescription(touch.button, options)},
-      {"altKey", getDebugDescription(touch.altKey, options)},
-      {"ctrlKey", getDebugDescription(touch.ctrlKey, options)},
-      {"shiftKey", getDebugDescription(touch.shiftKey, options)},
-      {"metaKey", getDebugDescription(touch.metaKey, options)},
+      {"button", getDebugDescription(touch.button, options)}, // [macOS]
+      {"altKey", getDebugDescription(touch.altKey, options)}, // [macOS]
+      {"ctrlKey", getDebugDescription(touch.ctrlKey, options)}, // [macOS]
+      {"shiftKey", getDebugDescription(touch.shiftKey, options)}, // [macOS]
+      {"metaKey", getDebugDescription(touch.metaKey, options)}, // [macOS]
```

Comment:
Not blocking, but for the future you can also just add start and end tags to a block of code. We have it documented here: https://github.com/microsoft/react-native-macos/blob/main/docs/DiffsWithUpstream.md 
```
// [macOS
...new code
// macOS]
```